### PR TITLE
Add rt crowd forecast

### PR DESCRIPTION
**Pull request by nikosbosse**
create functionality for an rt crowd forecast app

### CODE REVIEW

File: crowd-rt-forecast/app.R [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+# devtools::install_github("epiforecasts/forecasthubutils@update-make_weekly")
+# devtools::install_github("epiforecasts/crowdforecastr@rt-visualisation")
+options("golem.app.prod" = TRUE)
+
+# load submission date from data if on server
+if (!dir.exists("crowd-rt-forecast")) {
+  submission_date <- readRDS("data-raw/submission_date.rds")
+} else {
+  submission_date <- floor_date(Sys.Date(), unit = "week", week_start = 1)
```
Comment (seabbs) [inline crowd-rt-forecast/app.R:26]: latest_weekday?

File: crowd-rt-forecast/redeploy.R [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  mutate(target_type = "case", target_end_date = as.Date(target_end_date)) %>%
+  filter(target_end_date <= (as.Date(first_forecast_date) + 7 * 6)) %>%
+  arrange(region, target_type, target_end_date) %>%
+  rename(location_name = region) %>%
+  select(-strat, -type, -mean, -sd)
+
+fwrite(obs, here("crowd-rt-forecast", "data-raw", "observations.csv"))
+
+# copy Rt samples and fit data as well -----------------------------------------
+copyrt <- function(origin_dir, target_dir, locations, date) {
```
Comment (seabbs) [inline crowd-rt-forecast/redeploy.R:30]: call from somewhere just to make it easier to read

File: crowd-rt-forecast/redeploy.R [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    origin_folder <- here(origin_dir, location, date)
+    target_folder <- here(target_dir, location)
+    
+    if (dir.exists(origin_folder)) {
+      check_dir(target_folder)
+      file.copy(from = origin_folder, to = target_folder, recursive = TRUE)
+      
+      # read in epinow2 fit and thin
+      fit <- readRDS(here(target_folder, date, "model_fit.rds")) %>%
+        shredder::stan_slice(seq(1, 450, by = 450/25), inc_warmup = FALSE)
```
Comment (seabbs) [inline crowd-rt-forecast/redeploy.R:41]: I thought we diced to use the hacked version of the code as this package was a mess. Don't mind either way.

Comment (seabbs) [reply crowd-rt-forecast/redeploy.R:41]: I don't totally understand what the Rt model bits are only needed on redeployment but I assume it makes sense.

File: crowd-rt-forecast/redeploy.R [hunk 4]
```diff
@@ -0,0 +1,78 @@
+library(covid.ecdc.forecasts)
+library(data.table)
+library(dplyr)
```
Comment (seabbs) [inline crowd-rt-forecast/redeploy.R:3]: rstan missing

**Review by seabbs — APPROVED** [review]
LGTM

File: crowd-rt-forecast/redeploy.R (hunk 2 above)
Comment (nikosbosse) [reply crowd-rt-forecast/redeploy.R:30]: what do you mean? copying the function code somewhere else?

File: crowd-rt-forecast/redeploy.R (hunk 3 above)
Comment (nikosbosse) [reply crowd-rt-forecast/redeploy.R:41]: I wasn't able to get it done without shredder and that works. 
oh but need to add shredder to the dependencies...

**Pull request merged**